Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -28,6 +29,7 @@ public class ErrorPlugin : EventProcessorPluginBase {
             if (context.HasProperty("UserNamespaces"))
                 userNamespaces = context.GetProperty<string>("UserNamespaces").SplitAndTrim(',');
 
+            AddManualStackingKeyToSignatureIfExists(context);
```

Comment:
I think I may need a little help on this. Are you saying I should create another plugin which shouldl run before any other plugins (maybe right after 0_ThrottleBotsPlugin)? 
Or should I add a new action to the pipeline which would run before 005_RunEventProcessingPluginsAction?